gendo/bot.py

```diff
@@ -29,6 +29,8 @@ def __init__(self, slack_token=None, settings=None):
     def config_from_yaml(cls, path_to_yaml):
         with open(path_to_yaml, 'r') as ymlfile:
             settings = yaml.load(ymlfile)
+            log.info("settings from {path} loaded successfully".format(
+                     path=path_to_yaml))
```

Comment:
When you want to interpolate on log messages, pass variables in as arguments to the `log.info` function as per [Python logging docs](https://docs.python.org/3/howto/logging.html#logging-variable-data).

In this case either
```python
# 1. Pass interpolation variables as 
log.info('settings from %s loaded successfully', path_to_yaml)


# 2. or if you want to be more explicit
log_args = {'path': path_to_yaml}
log.info('settings from %(path)s loaded successfully', log_args)
```